source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,119 @@
+---
+title: oVirt Software Defined Networking, The OVN Network Provider
+author: mmirecki
+date: 2016-10-03 14:21:00 UTC
+tags: community, howto, blog, network
+comments: true
+published: true
+---
+
+oVirt offers not only its own internal networking, but also an API for external network providers. This API enables using external network management software inside environments managed by oVirt and takes advantage of their extended capabilities. 
+One of such solutions is OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that brings Software Defined Networking to OVS.
+
+OVN enables support for virtual networks abstraction by adding native OVS support for virtual L2 and L3 overlays.
+
+READMORE
+
+The oVirt provider for OVN consists of two parts:
+* The oVirt OVN driver
+* The oVirt OVN provider
+
+## oVirt OVN Driver
+
+The oVirt OVN driver is the Virtual Interface Driver placed on oVirt hosts that handles the wiring of VM NICs to OVN networking.
+
+The driver allows vdsm, libvirt, and OVN to interact whenever a NIC is pluged in such a way that the VM NIC is added to an appropriate OVN Logical Switch and the appropriate OVN overlays on all the hosts in the oVirt environment.
+
+The [oVirt OVN driver rpm](TODO: attach the RPM as late as possible to get the newest ones) is now available. The latest version can always be downloaded and built from the repository (described later in this article). Once the rpm is downloaded, it can be installed in the following way:
+
+    `dnf install ovn-provider-driver-0-1.noarch.rpm`
```

Comment:
I think a list of RPM package dependencies for OVN provider, for host and engine server roles is missing. This could aid in the installation.